<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parameter</class>
 <widget class="QWidget" name="parameter">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(238, 238, 236);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QGroupBox" name="server_groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(238, 238, 236);</string>
      </property>
      <property name="title">
       <string>服务器设置</string>
      </property>
      <widget class="QLabel" name="serverip_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>35</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>服务器IP：</string>
       </property>
      </widget>
      <widget class="QLabel" name="serverport_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>75</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>通讯端口：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="serverip_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>168</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="serverip_1">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>192</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="serverip_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="serverip_4">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="commport">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>190</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>50000</string>
       </property>
      </widget>
      <widget class="QLabel" name="par_tips">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>91</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 30pt &quot;Ubuntu&quot;;
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="client_groupBox">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(238, 238, 236);</string>
      </property>
      <property name="title">
       <string>本机设置</string>
      </property>
      <widget class="QLabel" name="client_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>35</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>客户端IP：</string>
       </property>
      </widget>
      <widget class="QLabel" name="date_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>75</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>时间设置：</string>
       </property>
      </widget>
      <widget class="QLabel" name="year_label">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>75</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>年</string>
       </property>
      </widget>
      <widget class="QLabel" name="month_label">
       <property name="geometry">
        <rect>
         <x>195</x>
         <y>75</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>月</string>
       </property>
      </widget>
      <widget class="QLabel" name="day_label">
       <property name="geometry">
        <rect>
         <x>255</x>
         <y>75</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>日</string>
       </property>
      </widget>
      <widget class="QLabel" name="hour_label">
       <property name="geometry">
        <rect>
         <x>315</x>
         <y>75</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>时</string>
       </property>
      </widget>
      <widget class="QLabel" name="min_label">
       <property name="geometry">
        <rect>
         <x>375</x>
         <y>75</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>分</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="clientip_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="clientip_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>168</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="clientip_4">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="clientip_1">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>192</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="year">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>2020</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="month">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>03</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="day">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>70</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>31</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="hour">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>24</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="min">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>70</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>24</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="comfirm_Button">
      <property name="text">
       <string>确认修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextpage_Button">
      <property name="text">
       <string>下一页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="return_Button">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="parameter1_title">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 36pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;参数设置1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
